docs/_website/src/theme/BlogPostItem/index.js

```diff
@@ -59,19 +59,19 @@ function BlogPostItem(props) {
           {/* Crazy avatar stack code */}
           <div style={{position: 'relative', height: "45px", width: 8 + 45 + ((authors.length - 1) * 20) + "px"}}>
           {authors.map(({ name, avatar }, idx) =>
-            <img className={styles.blogPostAvatar} style={{ zIndex: 1000 - idx, marginLeft: idx * 20 + "px" }} src={avatar} alt={name} />
+            <img key={name} className={styles.blogPostAvatar} style={{ zIndex: 1000 - idx, marginLeft: idx * 20 + "px" }} src={avatar} alt={name} />
           )}
           </div>
 
           <div className="avatar__intro">
             <h4 className={clsx(styles.blogPostAuthor, "avatar__name")}>
               {authors.map(({ name, url }, idx) =>
-                <>
+                <React.Fragment key={name}>
```

Comment:
I suppose a `React.Fragment` is purpose built for this type of issue?

Reply:
`<>` is shorthand for fragment, when appending a key you can't use the shorthand